README.in.rst

```diff
@@ -16,6 +16,16 @@
 .. |---| unicode:: U+2014  .. em dash, trimming surrounding whitespace
    :trim:
 
+Requirements
```

Comment:
`rbenv.el` should be gotten automatically, and I'm not completely sure rbenv is required.
bundler is required, and I will perhaps address this explicitly if this package gets any attention.  Thank you for your early interest.